src/plugins/upload.js

```diff
@@ -0,0 +1,52 @@
+import tus from 'tus-js-client'
+
+export default {
+  install(Vue) {
+    Vue.mixin({
+      data() {
+        return {
+          chunkedUploadFilesInProgress: []
+        }
+      },
+
+      methods: {
+        uploadChunkedFile(file) {
+          const upload = new tus.Upload(file, {
+            endpoint: this.$client.helpers._webdavUrl,
```

Comment:
I think you'll need to extract the path from the file name and append it to the endpoint.

as far as I understand the metadata should only contain the basename of the file name, not the full path

Reply:
Right, I'll adjust it :wink: